### PR TITLE
Bump BHC/activities

### PR DESCRIPTION
**Pull request by ryantmer**
Fixes change event dispatching in the instructions component of `d2l-activity-assignment-editor`.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3263,7 +3263,7 @@
         "fastdom": "^1.0.8",
         "merge-stream": "^1.0.1",
         "require-dir": "^1.2.0",
-        "siren-sdk": "github:BrightspaceHypermediaComponents/siren-sdk#semver:^1"
+        "siren-sdk": "github:BrightspaceHypermediaComponents/siren-sdk#semver:^1.4.2"
```
Comment (ryantmer) [inline package-lock.json]: FYI @jlai-d2l, this appears to come from [this change](https://github.com/BrightspaceHypermediaComponents/activities/pull/441/commits/728c68026bcbb0820f850e373e38c7a6652d4697#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R96) - seems okay, but wanted to verify that it's not a surprise.

File: package-lock.json [hunk 2]
```diff
@@ -853,9 +853,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.14.1",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.14.1.tgz",
-      "integrity": "sha512-q1jQi7uoSKO/AUHngn8647QIsptYP/F/81M0oX0F/LsOu+rnnI22y6R+r1cx+WiYopj9CQ7bM0df8JYRxJII9A==",
+      "version": "1.14.2",
```
Comment (ryantmer) [inline package-lock.json]: FYI @dlockhart

File: package-lock.json (hunk 1 above)
Comment (ryantmer) [reply package-lock.json]: Looks like this causes the build to fail, in fact - @jlai-d2l mind removing this from activities and doing a new release?

Comment (svanherk) [reply package-lock.json]: Yup, we need to keep this at `^1`, or else we'll get multiple versions of the sdk components which will cause registration errors due to name collisions

**Pull request merged**